plan/client.go

```diff
@@ -89,19 +73,7 @@ func (c Client) Update(id string, params *stripe.PlanParams) (*stripe.Plan, erro
 	if params != nil {
 		commonParams = &params.Params
 		body = &stripe.RequestValues{}
-
-		if len(params.Name) > 0 {
-			body.Add("name", params.Name)
-		}
-
-		if len(params.Statement) > 0 {
-			body.Add("statement_descriptor", params.Statement)
-		}
-
-		if params.TrialPeriod > 0 {
-			body.Add("trial_period_days", strconv.FormatUint(params.TrialPeriod, 10))
-		}
-
+		params.Params.AppendTo(body)
```

Comment:
Yeah that's a good point ... a major version bump seems to be in order.

> We may want to make this an explicit breaking change and have different types of PlanParams objects per method.

Yeah, that would probably be the better way to go. We should maybe consider just how impactful even a major version bump will be (they're all annoying, but some can be _really_ annoying) because that would be the kind of change where you'd have to update literally every invocation of Stripe that you have in your codebase. Maybe worth the risk though.